README.md

```diff
@@ -1,158 +1,270 @@
+        <li><a href="#installing-and-configuring-ignition">Installing and Configuring Ignition</a></li>
+        <li><a href="#restarting-ignition">Restarting Ignition</a></li>
+        <li><a href="#obtaining-an-ewons-name">Obtaining an Ewon's Name</a></li>
+      </ol>
+    </li>
+  </ol>
+
+  <!-- INFORMATION -->
+  <h2 id="information"><span style="font-variant: small-caps;">Information</span></h2>
+
+  <!-- INFORMATION/ABOUT -->
+  <h3 id="about"><span style="font-variant: small-caps;">About</span></h3>
+  <p>Ewon Ignition Demo is a project that showcases the features of integrating the HMS Networks’ Ewon platform and the Inductive Automation Ignition platform.</p>
+  <p>Ewon Ignition Demo was developed and is maintained by HMS Networks, using the Ewon Java library. For more information about this demo and/or developing applications with the Ewon Java library, please visit the Ewon Developer website at <a href="https://developer.ewon.biz/" class="uri">https://developer.ewon.biz/</a> or contact us at <a href="mailto:us-services@hms.se">us-services@hms.se</a>.</p>
+
+  <!-- INFORMATION/SYSTEM DESIGN -->
+  <h3 id="system-design"><span style="font-variant: small-caps;">System Design</span></h3>
+  <p>Ewon Ignition Demo is a two-part system consisting of an Ewon application and an Ignition Designer project.</p>
+  <p>The Ewon-side application uses FTP to configure tags on the Ewon, then creates simulated tag data and applies it to the Ewon’s tags using the Ewon Java library.</p>
+  <p>The Ignition Designer project, in conjunction with the Ewon Ignition Connector, accesses the data created by the demo’s Ewon application, and displays it in realistic dashboards that are designed to mimic those of a real system.</p>
+
+  <h3 id="trademark-information"><span style="font-variant: small-caps;">Trademark Information</span></h3>
+  <p><small>Ewon&reg; is a registered trademark of HMS Industrial Networks SA. Microsoft&reg; and Windows&reg; are registered trademarks of Microsoft Corporation in the United States and/or other countries. Inductive Automation&reg; is a registered trademark of Inductive Automation, LLC. All other trademarks mentioned in this document are the property of their respective holders.</small></p>
+
+  <!-- GETTING STARTED -->
+  <h2 id="getting-started"><span style="font-variant: small-caps;">Getting Started</span></h2>
+
+  <!-- GETTING STARTED/PREREQUISITES -->
+  <h3 id="prerequisites"><span style="font-variant: small-caps;">Prerequisites</span></h3>
+  <p>These are the items that need to be ready or configured prior to installation and use of the Ewon Ignition Demo.</p>
+  <ul>
+    <li>
+      <p><strong>Ewon Device</strong></p>
+      <ul>
+        <li>
+          <p>For information on how to setup and configure your Ewon device, please refer to <em><a href="#setting-up-an-ewon">Appendix 6.1, Setting Up an Ewon</a>.</em></p>
+        </li>
+      </ul>
+    </li>
+    <li>
+      <p><strong>PC (Laptop/Desktop)</strong></p>
```

Comment:
Why do you need to call out Laptop/Desktop?  Technically this could be a tablet, thin client, raspberry pi (whatever category they fall under), etc.  It just really needs a version of windows, linux, macos that ignition supports.

Reply:
changed